Dockerfile

```diff
@@ -87,14 +87,13 @@ ENV HOME /home/app
 #####
 
 # TODO: use packrat (or something else) for R package management
-COPY packages.R $HOME
-RUN Rscript packages.R
-
-# install custom packages from R/pkgs/**
-COPY local_install.R $HOME
-COPY R/pkgs $HOME/R/pkgs
-RUN Rscript local_install.R
-
+RUN Rscript -e "install.packages('packrat',repos='https://cloud.r-project.org/')"
+RUN Rscript -e "install.packages('arrow',repos='https://cloud.r-project.org/')"
+RUN Rscript -e 'arrow::install_arrow()'
```

Comment:
should be 1 `RUN` statement cojoined with `&&`. docker creates a new layer per RUN command, which is semi-expensive, so we should make them compact.